ddtrace/tracer.py

```diff
@@ -490,7 +490,7 @@ def _start_span(
             if isinstance(child_of, Context):
                 context = child_of
             else:
-                context = child_of.context
+                context = child_of._context
```

Comment:
if `_context` is `None` here, does that mean these two spans will be linked in the same trace, but will not share the same context?

what would happen with origin and sampling priority?

Reply:
Yes but the context references were copied before as well. Spans don't share the same context but rather we make copies of the context (if available) in order to have the spans share the trace-level tags